### ISSUE TITLE
Keybindings

### ISSUE THREAD

**Issue by rornor** [darikg/PgcliSublime#15]
Hi,

this plugin overwrites default ST bindings for find all/replace all.

IMHO it would be more sensitive to provide bindings that does not overwrite ST shortcuts.

**Comment by darikg** [comment]
You're absolutely right, thanks for pointing that out. Any suggestions on what default bindings to provide that don't override ST defaults?

**Comment by rornor** [comment]
I have no idea. Maybe you can just limit the binding to sql in lack of inspiration :)
Something like this:

```
  {
    "keys": ["alt+enter"], 
    "command": "pgcli_run_all",
    "context": [
      {"key": "selector", "operand": "source.sql"}
    ]
  },
```

**Comment by darikg** [comment]
Perfect, thanks! Fixed in v0.1.8
